aries-site/src/data/wcag/components.json

```diff
@@ -862,6 +862,518 @@
+        "notes": ""
+      },
+      {
+        "rule": "2.3.2",
+        "label": "Three Flashes",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.3.3",
+        "label": "Animation from Interactions",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.1",
+        "label": "Bypass Blocks",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.2",
+        "label": "Page Titled",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.3",
+        "label": "Focus Order",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.4",
+        "label": "Link Purpose (In Context)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.5",
+        "label": "Multiple Ways",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.6",
+        "label": "Headings and Labels",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.7",
+        "label": "Focus Visible",
+        "status": "failed",
+        "notes": "TextInput with suggestions, the treatment of the suggestion that is in focus is not meeting requirement."
+      },
+      {
+        "rule": "2.4.8",
+        "label": "Location",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.9",
+        "label": "Link Purpose (Link Only)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.10",
+        "label": "Section Headings",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.11",
+        "label": "Focus Not Obscured (Minimum)",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.12",
+        "label": "Focus Not Obscured (Enhanced)",
+        "status": "failed",
+        "notes": "TextInput with suggestions, the treatment of the suggestion that is in focus is not meeting requirement."
+      },
+      {
+        "rule": "2.4.13",
+        "label": "Focus Appearance",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "2.5.1",
+        "label": "Pointer Gestures",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.5.2",
+        "label": "Pointer Cancellation",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "2.5.3",
+        "label": "Label in Name",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "2.5.4",
+        "label": "Motion Actuation",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.5.5",
+        "label": "Target Size (Enhanced)",
+        "status": "passed",
```

Comment:
Doesn't look like we are meeting 44x44 target size

Reply:
Maybe I should put "conditional"? Because according to WCAG on target size, if there's enough space around it to meet that dimension, then it passes. Our textinput is 36px tall and wider than 44px and most likely always has 4px above/below so it would pass.

What do you think? "passed" or "conditional"?

Reply:
I'm not seeing the 'space around' exception on the Target Size (Enhanced) rule. I do see if on the Target Size (minimum) rule. Could you point me in the right direction to where you are seeing this, I might be missing something?

Reply:
I'm not seeing the 'space around' exception on the Target Size (Enhanced) rule. I do see if on the Target Size (minimum) rule. Could you point me in the right direction to where you are seeing this, I might be missing something?

Reply:
Ah I misread and was reading the AA rule instead which allows for space around. Updating.